testsuite/src/it/java/io/quarkus/ts/startstop/CodeQuarkusTest.java

```diff
@@ -93,7 +93,7 @@ public void testRuntime(TestInfo testInfo, List<CodeQuarkusExtensions> extension
             appendln(whatIDidReport, "# " + cn + ", " + mn);
             appendln(whatIDidReport, (new Date()).toString());
             LOGGER.info("Downloading...");
-            appendln(whatIDidReport, "Download URL: " + download(extensions, zipFile));
+            appendln(whatIDidReport, "Download URL: " + download(extensions, zipFile, 11));
```

Comment:
In the main branch, we could create a new test like `java17BasedProject` but pointing to JDK11, because the rest of the scenarios are going to be JDK17 by default. Indeed the test `java17BasedProject` could be renamed to `defaultJDKProject` in order to check that the default JDK is set to 17.

Reply:
https://github.com/quarkus-qe/quarkus-startstop/pull/219 created to address this